tools/Version.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "Collector": "2.0.7",
+    "Collector": "2.0.8",
     "Analyzer": "2.0.6",
     "Generator": "2.0.4"
```

Comment:
Good point, idk

Reply:
In my opinion, we just bump it every time we make a change.

Or at least establish a standard for what determines what signifies a new release version for consistency.

Reply:
I agree, originally the idea of standard I was using was:

2 (major releases). 0 (minor releases, i.e: new sprint). 5 (any change, patch and bugfixes)

Bug then with Github version control, we stopped to bump it with minor changes....

But I think is valid to bump it with any change as it might improve the outcome of the JSON, Excel and Reports...

Let's move with that approach?

Reply:
The new sprint might be difficult to keep consistent. Let's just use the standard release versioning:

    MAJOR version increases for incompatible changes,
    MINOR version increases for backward-compatible changes, and
    PATCH version increases for backward-compatible bug fixes.